main.go

```diff
@@ -272,9 +273,16 @@ func main() {
 			log.Printf("Forcing update of inputTable: %s", inputConf.Table)
 		}
 
-		fatalIfErr(runCopy(db, *inputConf, *inputTable, targetTable, *truncate, *gzip, *delimiter, *timeGranularity), "Issue running copy")
-		// DON'T NEED TO CREATE VIEWS - will be handled by the refresh script
-		log.Printf("done with table: %s.%s", inputConf.Schema, t)
+		if err := runCopy(db, *inputConf, *inputTable, targetTable, *truncate, *gzip, *delimiter, *timeGranularity); err != nil {
```

Comment:
I was actually a big fan of https://github.com/hashicorp/go-multierror when I last needed something similar to this. It might make it a little simpler, but your call.